### PR TITLE
xView2: deterministic dataset order

### PR DESCRIPTION
**Pull request by adamjstewart**
This is the only other place I found where we're using a set. @RitwikGupta maybe you can confirm that dataset order is not deterministic (even with setting a random seed) and confirm that this fixes it?

### CODE REVIEW

**Comment by isaaccorley** [comment]
I think for this case this isn't as big of an issue because the train/test sets are in separate folders. But if you did random split the train set into a train/val set you would encounter issues with determinism due to the set operation.

**Review by isaaccorley — APPROVED** [review]

**Comment by adamjstewart** [comment]
You still have issues with determinism even without random split. The order you iterate over the dataset changes, so you can't reproduce training results.

**Comment by isaaccorley** [comment]
Train dataloaders random shuffle by default on each epoch, but yeah if you want to guarantee that that randomness is identical per seed, this definitely needs to be fixed.
